package gamer.cfg;
import "common_cfg.proto";

message CameraCfg {
  required Vec3 camera_pos     = 1;
  required Vec3 camera_lookat  = 2;
}